backend/src/routes/ride/request.ts

```diff
@@ -7,16 +7,16 @@ import { RideOffer } from "../../database/entity/RideOffer";
 import { RideRequest } from "../../database/entity/RideRequest";
 import { User } from "../../database/entity/User";
 import { RequestToOffer } from "../../database/entity/RequestToOffer";
+import { subMinutes, addMinutes } from "date-fns";
+import { getOSRMDuration } from "../../middleware/osrmduration";
 import { AppDataSource } from "../../data-source";
 import { AppError } from "../../util/error-handler";
-import { parseJSON, subMinutes, addMinutes } from "date-fns";
-import { getOSRMDuration } from "../../middleware/osrmduration";
 
 export const postRideRequests = async (req: Request, res: Response) => {
-  const { email, offers, src, dest, startTime, window, distance, price } =
+  const { userID, offers, src, dest, startTime, window, distance, price } =
```

Comment:
We don't need the user id right? The auth guard sets the user object to res.locals.user so we won't have to fetch it again here

Reply:
Removed it

Reply:
Removed additional param in postRideRequests